_component_design/accessibility.md

```diff
@@ -0,0 +1,219 @@
+
+Provide feedback for interactions, such as confirming form submission, alerting the user when something goes wrong, or notifying the user of changes on the page. Instructions should be easy to identify and suggestions to help fix issues whenever errors are returned must be provided. Important feedback that requires a user action should be presented in a prominent style.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Failed text inputs
+
+There are many acceptable ways to make text inputs visually accessible. If you’re indicating a failed text input, make sure to include multiple visual indicators. For example, in addition to the ‘error message’, the border color turns red and doubles in thickness from 1px to 2px for better accessibility.
+
+It is important to reserve the prominent border at 2px for failed text input as an additional visual indicator for error state.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Default state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-1.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Error state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-2.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Focus for Input Field
+
+In this example, we provide a clear visual feedback that the focus has moved to the input field by changing the border color and adding a highlight to give the user clear indication of where they’re currently taking action.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Default state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-3.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [class:"image bg-light border"] %}
+![caption:"Error state"]({{site.url}}/assets/images/overview/accessibility/accessibility-inputs-4.svg)
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Tools
+
+These best practices are not exhaustive, and they are not necessarily applicable to every situation. However, they do cover the major areas for concern when designing for visual accessibility. It is more important to take away the principles, so that you can integrate them into your design processes. Ultimately, improving accessibility benefits all users by providing greater ease of use to everyone because well-designed products are both more usable and more accessible.
```

Comment:
principles so that you can* (remove comma) 

"Ultimately, improving accessibility benefits all users by providing greater ease of use to everyone." (remove rest of sentence)

Reply:
Updated.